Sources/Core/Presenter/Channel/ChannelPresenter+Request.swift

```diff
@@ -91,7 +91,7 @@ extension ChannelPresenter {
                 return
             }
             
-            if parentMessage == nil, message.parentId != nil {
+            if parentMessage == nil, message.parentId != nil, message.showReplyInChannel == false {
```

Comment:
Why not `!message.showReplyInChannel`?

Reply:
I actually prefer `== false`, too. `!` is harder to spot for me,

Reply:
@b-onc @VojtaStavik we have to get the agreement for these cases. I use `== false` really rare, mostly for readability in complex conditions.

Reply:
My vote is for `!`